js_modules/dagster-ui/packages/ui-core/src/assets/useWipeAssets.tsx

```diff
@@ -21,6 +27,14 @@ export function useWipeAssets({
     ASSET_WIPE_MUTATION,
     {refetchQueries},
   );
+  const [requestBackgroundAssetWipe] = useMutation<
+    BackgroundAssetWipeMutation,
+    BackgroundAssetWipeMutationVariables
+  >(BACKGROUND_ASSET_WIPE_MUTATION, {refetchQueries});
+  const [
+    getBackgroundWipeStatus,
+    {error: backgroundWipeStatusError, data: backgroundWipeStatusData},
```

Comment:
This object should also include functions `startPolling` and `stopPolling` that you can use. You might not need `startPolling` since you can provide the `pollInterval` at the callsite, but I think you will want to explicitly stop polling when the wipe has succeeded or failed.